webapp/lib/charts.dart

```diff
@@ -160,6 +166,9 @@ class DailyTimeseriesLineChartView {
         ..addAll(timeseriesPoints);
     }
     chart.options.scales.xAxes[0].time = (new chartjs.TimeScale(unit: timeScaleUnit));
+    if (timeScaleUnit == 'hour') {
+      chart.options.scales.xAxes[0].ticks = (new chartjs.LinearTickOptions()..maxTicksLimit = 21);
```

Comment:
consider using `stepSize: 2` instead of setting `maxTicksLimit` - I think it makes it a bit clearer that we want to show every two hours.
You might need to also add in an else clause to reset the ticks, e.g.:
```suggestion
      chart.options.scales.xAxes[0].ticks = (new chartjs.LinearTickOptions()..stepSize = 2);
    } else {
      chart.options.scales.xAxes[0].ticks = (new chartjs.LinearTickOptions());
```